src/context/AppContext.js

```diff
@@ -104,7 +90,18 @@ const AppProvider = ({children}) => {
 
   const updateEvents = (newEvent) => {
     setEvents([...events, newEvent])
-    // helperFunction({input: newEvent})
+    const { tourId, name, venueId, startTime, endTime } = newEvent
+    mutateEvent({
+      variables: {
```

Comment:
This makes perfect sense. I think if we hadn't already been working for hours yesterday we would have seen this.